OpenTabletDriver.Configurations/Configurations/Gaomon/M7.json

```
@@ -0,0 +1,41 @@
+{
+  "Name": "Gaomon M7",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 258.465,
+      "Height": 161.535,
+      "MaxX": 51693,
+      "MaxY": 32307
+    },
+    "Pen": {
+      "MaxPressure": 16384,
```

double check this, but very likely the actual max is:
```suggestion
      "MaxPressure": 16383,
```